src/UI/CardSelector.hs

```diff
@@ -57,7 +57,13 @@ drawUI s =
   [ drawMenu s <=> drawException s ]
 
 title :: Widget Name
-title = withAttr titleAttr $ hCenteredStrWrap "Select a deck of flashcards"
+title = withAttr titleAttr $ str "Select a deck of flashcards "
+
+shuffledWidget :: State -> Widget Name
+shuffledWidget s = withAttr exceptionAttr $ str $ 
+    case s^.gs.doShuffle of
+      True -> "(Shuffled)"
+      False -> ""
```

Comment:
```suggestion
    if s^.gs.doShuffle then "(Shuffled)" else ""
```
It's a bit clearer to do an if-then-else instead of a case distinction when you have boolean values.